engines/mtropolis/detection_tables.h

```diff
@@ -620,14 +671,65 @@ static const MTropolisGameDescription gameDescriptions[] = {
 			},
 			Common::DE_DEU,
 			Common::kPlatformWindows,
-			ADGF_UNSTABLE,
+			ADGF_UNSUPPORTED,
 			GUIO0()
 		},
 		GID_ALBERT2,
 		0,
 		MTBOOT_ALBERT2_WIN_DE,
 	},
 
+	{ // Uncle Albert's Fabulous Voyage (English, Windows)
+	  // Published by Emme, 2000
+		{
+			"albert2",
+			"",
+			{
+				{"voyage.exe",  0, "0e513dac9d2a0d7cfcdc670cab2a9bda", 757760},
+				{"voyage1.MPL", 0, "1859b9507c118f52402ecb248fa4ae9c", 25972983},
+				{"voyage2.MPX", 0, "639428214b71e057c825f41402d3694e", 254333940},
+				// { "BASIC.X95",    0, "a1b474e90cc285bfdcfa87355f890b9e", 242688 },
+				// { "BITMAP.R95",   0, "2bf7305eca9df63dbc75f55093b84cc0", 81408 },
+				// { "CURSORS.C95",  0, "c1e2eebe7183b5ea33de7872a8ea4d2b", 87040 },
+				// { "EXTRAS.R95",   0, "c5830771609c774de9dbeaa5dff69b3e", 185344 },
+				// { "ROTATORK.R95", 0, "c271f786f028c0076635bea49a2be890", 66560 },
+				AD_LISTEND
+			},
+			Common::EN_ANY,
+			Common::kPlatformWindows,
+			ADGF_UNSUPPORTED,
+			GUIO0()
+		},
+		GID_ALBERT2,
+		0,
+		MTBOOT_ALBERT2_WIN_EN,
+	},
+
+	{ // Uncle Albert's Fabulous Voyage (French, Windows)
+	  // Original title: Le Fabuleux Voyage de l'oncle Ernest
+	  // Published by Emme, 1999
+		{
+			"albert2",
+			"",
+			{
+				{"voyage.exe", 0, "0e513dac9d2a0d7cfcdc670cab2a9bda", 757760},
+			 {"voyage1.MPL", 0, "174683d29a1c8511830262940223cc8a", 27889879},
```

Comment:
Same problem with indentations. It looks that spaces sneaked in instead of tabs.

Reply:
I went through the file again and fixed the whitespaces deviations.